subscription-contracts/src/LaunchPadApprovalHook.sol

```diff
@@ -33,12 +36,20 @@ contract LaunchPadApprovalHook is IJBRulesetApprovalHook {
         return 0;
     }
 
+    function enableRefunds(bool _refundsEnabled) external onlyOwner {
+        refundsEnabled = _refundsEnabled;
+    }
+
     function approvalStatusOf(
         uint256 projectId,
         JBRuleset memory ruleset
     ) external view override returns (JBApprovalStatus) {
         uint256 currentFunding = _totalFunding(terminal, projectId);
-        if (currentFunding >= fundingGoal && block.timestamp >= deadline) {
+        if (refundsEnabled && block.timestamp < deadline + refundPeriod) {
```

Comment:
Can you ELI5 for me? This logic is a little tricky so I want to 100% make sure I wrap my head around it.

I'm a little confused what JBApprovalStatus.Approved / Failed is doing. Does this turn contributions on or off?